core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosClusterScheduler.scala

```diff
@@ -422,6 +422,37 @@ private[spark] class MesosClusterScheduler(
     desc.schedulerProperties.get("spark.cores.max").map { v =>
       options ++= Seq("--total-executor-cores", v)
     }
+
+    /**
+      * The properties to pass along from Spark Submit to the Mesos Scheduler.
+      * http://spark.apache.org/docs/1.5.1/configuration.html#available-properties
+      */
+    val propertiesToPropagate = Set(
+      "spark.driver.extraClassPath",
```

Comment:
yeah why not just ship everything? If we add a config in the future but forget to add it here this will fail in mysterious ways.